### ISSUE TITLE
Backup via services and non-ANSI path for output log file

### ISSUE THREAD

**Issue by shavluk** [FirebirdSQL/firebird#7490]
Creating a backup via services (-se) and outputting the log-file (switch "-y") with a path that contains non-ANSI characters ends with an error. 
Tested on 3.0.10 and 4.0.2 (Win10 x64)

`gbak.exe -b -user SYSDBA -password masterkey -se localhost:service_mgr -y d:\Тест\test.log d:\db\test.fdb d:\db\test.fbk`

If log-file path contains non-ANSI characters, there is an error. The error is repeated on 3.0.10 and 4.0.2
`gbak: ERROR:cannot open status and error output file d:\РўРчС?С'\test.log`

**Comment by aafemt** [comment]
Non-ANSI paths and filenames are not supported by Firebird at all.

**Comment by shavluk** [comment]
> Non-ANSI paths and filenames are not supported by Firebird at all.

Non-ANSI paths for database and backup it works

`gbak.exe -b -user SYSDBA -password masterkey -se localhost:service_mgr d:\Тест\test.fdb d:\Тест\test.fbk`
without errors

**Comment by aafemt** [comment]
This ticket should say "non-ASCII" characters.
